modules/exploits/user_access_manager_reflected_xss_shell_upload.rb

```diff
@@ -7,6 +7,7 @@ def initialize
     update_info(
       name: 'User Access Manager <= 2.0.0 Reflected XSS Shell Upload',
       author: [
+        'http://www.defensecode.com/'             # Dislosure
```

Comment:
Can you attribute DefenseCode as "DefenseCode" rather than the URL? Also needs a comma adding to correct the syntax error.